src/main/java/org/italiangrid/storm/webdav/tpc/http/HttpTransferClient.java

```diff
@@ -237,6 +236,14 @@ public void handle(GetTransferRequest request, TransferStatusCallback cb) {
       if (!reportTask.isCancelled()) {
         reportTask.cancel(true);
       }
+      if (resolver.resolveStorageArea(request.path()).tapeEnabled()) {
+        try {
+          attributesHelper.setPremigrateAttribute(filePath);
+        } catch (IOException e) {
+          LOG.warn(
+              "Error setting premigrate extended attribute to {}: {}", filePath, e.getMessage(), e);
```

Comment:
There are three arguments for only two placeholders in the format string. Is it correct?

Reply:
And do we need the stack trace here?